### PR TITLE
Update project name throughout the repository

### PR DESCRIPTION
**Pull request by lbianchi-lbl**
## Summary/Motivation

- Replace project name everywhere so that everything works with new project name

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by lbianchi-lbl** [comment]
I needed to work a bit to set up ReadTheDocs, but it looks like it's working now (even though we might have to wait until this PR is merged in to have the PR check show up in the PR Checks menu): https://watertap--234.org.readthedocs.build/en/234/index.html.

If you don't have anything against, I'd say we can go ahead and merge this in, and deal with any remaining fixes in separate PRs.

**Review by bknueven — APPROVED** [review]
The tests pass, and a cursory `git grep` for proteuslib and related names comes up empty locally.

**Review by ksbeattie — APPROVED** [review]
LGTM

**Pull request merged**
